include/rocksdb/statistics.h

```diff
@@ -226,6 +226,39 @@ enum Tickers : uint32_t {
   // Number of internal keys skipped by Iterator
   NUMBER_ITER_SKIP,
 
+  // BlobDB specific stats
+  BLOB_DB_NUM_PUT,
```

Comment:
I wonder if there is a way to put all these new BlobDB tickers and histograms inside blob_db directory itself close to other blob related code? 
I am just thinking of ways to avoid polluting this file with Blob related metrics; but am open to it if there is no alternative. Can the fb stats collector collect metrics by looking at header files? I wonder how other stackable dbs are exporting their metrics.

Reply:
I did a quick search and no stackable db currently have statistics. Blob db is the first one. I discussed with @ajkr and we don't have good way to separate blob db stats. Let me know if you have any suggestions.

Reply:
ok. thanks for checking.